### PR TITLE
Fix margins and arrangement of the activity tab header

### PR DESCRIPTION
**Pull request by b12f**
Closes https://community.openproject.org/projects/openproject/work_packages/39771/activity

### CODE REVIEW

File: frontend/src/global_styles/layout/work_packages/_details_view.sass [hunk 1]
```diff
@@ -83,52 +83,69 @@ body.router--work-packages-partitioned-split-view-new
...
+.work-package-details-tab
+  overflow-y: auto
+  overflow-x: hidden
+  flex-grow: 1
+  padding: 0 5px 10px 20px
+  @include styled-scroll-bar
+
+.work-packages--breadcrumb
+  display: grid
+  grid-template-columns: 1fr auto
```
Comment (HDinger) [inline frontend/src/global_styles/layout/work_packages/_details_view.sass]: This does not make sense any more as you change the html structure, and the button is not part of the breadcrumb

**Review by HDinger — CHANGES_REQUESTED** [review]
I have some remarks:
* The icons in the tabs are not aligned with the text
Bildschirmfoto 2021-11-10 um 12 58 01

* In the specification the tabs have more space to the top, than it is currently implemented.

Further, this has huge conflicts with #9830 , but I think makes sense to merge this one here first, and adapt the other one later..

File: frontend/src/global_styles/layout/work_packages/_details_view.sass (hunk 1 above)
Comment (b12f) [reply frontend/src/global_styles/layout/work_packages/_details_view.sass]: Oops, true

**Comment by b12f** [comment]
> The icons in the tabs are not aligned with the text 

I only touched the stuff below the tab titles in this PR, I'd open up a second PR if you want to see that fixed. Perhaps as a part of or after https://github.com/opf/openproject/pull/9775

**Review by HDinger — APPROVED** [review]
> I only touched the stuff below the tab titles in this PR, I'd open up a second PR if you want to see that fixed. Perhaps as a part of or after #9775

It's fine for me to do it separately, probably it is better to do it after #9775 so that we get this finished. 

Waiting for the tests now. When they are green, it is good to merge from my side

**Pull request merged**